app/src/components/dialog/attachments/AttachmentTypeSelector.tsx

```diff
@@ -26,9 +26,17 @@ const AttachmentTypeSelector: React.FC<IAttachmentTypeSelectorProps> = (props) =
     return <></>;
   }
 
+  const instanceOfIGetProjectAttachment = (object: any): object is IGetProjectAttachment => {
+    return true;
+  };
+
+  const instanceOfIGetSurveyAttachment = (object: any): object is IGetSurveyAttachment => {
+    return true;
+  };
+
```

Comment:
Should probably check something about `object` here to confirm what type it is.

Something like:
```
return !!object['project_attachment_id']
```

Should also define a type rather than just `any`.